pkg/scheduler/plugins/predicates/cache.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
s/Kubernetes/Volcano

Reply:
Modified  -- Copyright 2020 The Volcano Authors.